codecov.yml

```diff
@@ -0,0 +1,8 @@
+coverage:
+  status:
+    project:
+      default:
+        threshold: 0.02%
+    patch:
+      default:
+        target: 100%
```

Comment:
I know we would like that and we tend to enforce it but making it a strict requirement makes me nervous.

Reply:
It won't break the build. I.e., we can still merge something if we want to, but I think having it fail for any diff that doesn't have 100% coverage makes sense, so that we are sure to be alerted and have the opportunity to either fix it or ignore it.

Reply:
Isn't it still marking the commit as "failed"?

Reply:
Yes, it is, but it doesn't prevent us from merging it. If we are aiming at 100% coverage for any new code landing, then I don't really see a point in setting this to anything other than 100%. We could set it to 90%, or something like that, but then what's the point? If someone tries to land something with 91% code coverage we'd still want that flagged, wouldn't we? The number of times that someone is going to try to land code without 100% coverage *and* we think that's ok is going to be very small, imo. I'd rather be alerted to those rare circumstances by a failing check than not.

Reply:
Isn't this piece of information in the codecov comment?

I don't think we should have "red" commits in the history, which will happen with this check.

Reply:
Ok, but that doesn't address my issue of what is the point of having this check at all, if it's going to pass when people try to land code with missing coverage? Maybe we should just disable this `patch` check entirely? We can set the target to 80 or 90%, but that's really just making an arbitrary guess about what might pass that we think is okay.

Reply:
> Maybe we should just disable this `patch` check entirely?

Yes, I also think we should disable it. Sorry, I thought that was clear.

Reply:
It was not 😄, but I'll give it a try

Reply:
Disabled!